### ISSUE TITLE
[Bug] v-update-user-package error hestiamail

### ISSUE THREAD

**Issue by arielesp** [hestiacp/hestiacp#4101]
### Describe the bug

a clean hestiacp 1.8.10 install

error in new user hestiamail, but he user exists in /etc/passwd

in the panel, user count show 2 , but only admin exists

grep: /usr/local/hestia/data/users//hestiamail/user.conf: No such file or directory
Error: user hestiamail doesn't exist

### Tell us how to replicate the bug

# v-update-user-package default
grep: /usr/local/hestia/data/users//hestiamail/user.conf: No such file or directory

# v-update-sys-ip
sed: can't read /usr/local/hestia/data/users/hestiamail/web.conf: No such file or directory
Error: user hestiamail doesn't exist

# cat /etc/passwd
hestiamail:x:1001:1001:email@email.com:/home/hestiamail:/bin/sh

### Which components are affected by this bug?

Control Panel Command Line Interface
Control Panel Interface

### Hestia Control Panel Version

1.8.10

### Operating system

ubuntu 22.04

**Comment by jaapmarcus** [comment]
$BIN/v-list-sys-users plain

Check /etc/passwd for @ ... Will fix the issue for next release. It has been already patched for the main branch

**Comment by arielesp** [comment]
> $BIN/v-list-sys-users plain
> 
> Check /etc/passwd for @ ... Will fix the issue for next release. It has been already patched for the main branch

hestiamail
admin

in /etc/paswd not exists "@"

thx

**Comment by jaapmarcus** [comment]
eris@web01:~$ cat /etc/passwd | grep '@'
```
hestiamail:x:1000:1000:me@xxxx.xx:/home/hestiamail:/bin/sh
admin:x:1001:1001:me@xxx.:/home/admin:/usr/sbin/nologin
zzzz:x:1003:1003:client@domain.com:/home/zzzz:/bin/bash
```

So that is the issue issue has been patched in the main branch but not in the release branch yet ...

**Comment by arielesp** [comment]
yes, there are these users ..... @ is my email from install in hestiamail and admin , like your example

OK thanks =)
